docs/features/updating-tags.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Running Inference"
```

Comment:
"Updating Tags"

Reply:
Actually looks like this file was copied over but retains the content on running inference?

Reply:
Yep, I haven't written any of the docs yet